lib/graphql/static_validation/rules/argument_literals_are_compatible_error.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+module GraphQL
+  module StaticValidation
+    class ArgumentLiteralsAreCompatibleError < Message
```

Comment:
Perhaps this is something we can address in a subsequent PR, but I wonder if we should rename `StaticValidation::Message` to `StaticValidation::Error`.

Reply:
I can do it in this PR if people would prefer. It's all related. Any opinion @rmosolgo ?

Reply:
I've just gone ahead and done it!